en/services/heartbeat.md

```diff
@@ -1,21 +1,45 @@
 # Heartbeat/Connection Protocol
 
-The heartbeat protocol is used to determine whether a system is connected, and to detect when it has disconnected.
+The heartbeat protocol is used to advertise the existence of a system on the network, along with its system and component id, vehicle type, flight stack, component type, and flight mode.
 
-## Connection/Disconnection
+The heartbeat allows other components to:
+- detect when a system connects/disconnects to the network. A system is considered "active" (alive) if its [HEARTBEAT](../messages/common.md#HEARTBEAT) message is regularly received (and inactive if expected messages are not received).
```

Comment:
What I don't like about "active" and "inactive" is that you infer whether a system is active from the heartbeats that you receive but actually a system can be active even if you don't receive heartbeats, it's just that you have not discovered the system. So said differently "active" describes the state of the system when actually all you know is the state of the system seen through a transport layer.

In the SDK I chose to use the word "connection" for the transport layer (e.g. serial or UDP) and "discovered" vs. "timed out" for the system state. It's not optimal either and can be confusing but I consider it better than "active"/"inactive". Maybe "discovered"/"not discovered" would be better. 

Also "active"/"inactive" can be confused with the "armed"/"disarmed" state.

Reply:
@julianoes That's a reasonable point thanks. Discovered and timed out (or similar) might work. I'll look at this tomorrow. 
I just need to differentiate between something that we observe on the network vs something that we are actively communicating with.